azure-kusto-ingest/azure/kusto/ingest/_resource_manager.py

```diff
@@ -140,3 +140,6 @@ def retrieve_service_type(self):
             return command_result.primary_results[0][0][_SERVICE_TYPE_COLUMN_NAME]
         except (TypeError, KeyError):
             return ""
+
+    def set_proxy(self, proxy_url: str):
```

Comment:
Better to just create a new ResourceManager

Reply:
The change is only in the kusto client, and since I don't think we want to create a new client (since we don't want to restart the token provider), creating a new resource manager with the same client wouldn't change anything.

Reply:
I'm mean create the ResourceManager with the modified client instead of exposing a way to set a property of a member